src/host-config/index.ts

```diff
@@ -0,0 +1,132 @@
+import { promises as fs } from 'fs';
+import { isRight } from 'fp-ts/lib/Either';
+import Reporter from 'io-ts-reporters';
+
+import { disallowedProxyFields } from './proxy';
+import type { RedsocksConfig, ProxyConfig } from './types';
+import { HostConfiguration, LegacyHostConfiguration } from './types';
+import { readProxy, setProxy } from './proxy';
+import * as config from '../config';
+import * as applicationManager from '../compose/application-manager';
+import { pathOnRoot } from '../lib/host-utils';
+import log from '../lib/supervisor-console';
+import * as updateLock from '../lib/update-lock';
+
+export * from './proxy';
```

Comment:
Exporting all the submodules is so that the rest of the Supervisor sees host-config as one submodule entity - there won't be a need to `import X from './host-config/proxy'`, for example.

Reply:
But why does the rest of the supervisor need direct access to the proxy module? The interface to interact with the proxy is this module

Reply:
Removed unnecessary exports in separate commit https://github.com/balena-os/balena-supervisor/commit/16d8a59f9c185fcef1a08eab5fb6f156b4763d6b